diff --git a/scripts/posix/start.sh b/scripts/posix/start.sh
index d444ad77..660c075b 100755
--- a/scripts/posix/start.sh
+++ b/scripts/posix/start.sh
@@ -1,14 +1,10 @@
 #!/bin/bash
 
-if grep -qi 'arch' /etc/os-release || command -v pacman &> /dev/null; then
-    STEAM_PATH="/usr/lib/steam/steam"
-else
-    STEAM_PATH="/usr/lib/steam/bin_steam.sh"
-fi
+STEAM_PATH="@steam@/bin/steam"
 
 export OPENSSL_CONF=/dev/null
-export LD_PRELOAD="/usr/lib/millennium/libmillennium_x86.so${LD_PRELOAD:+:$LD_PRELOAD}" # preload Millennium into Steam
-export LD_LIBRARY_PATH="/usr/lib/millennium/${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+export LD_PRELOAD="@OUT@/lib/millennium/libmillennium_x86.so${LD_PRELOAD:+:$LD_PRELOAD}" # preload Millennium into Steam
+export LD_LIBRARY_PATH="@OUT@/lib/millennium/${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
 
 # Millennium hooks __libc_start_main to initialize itself, which is a function that is called before main. 
 # Besides that, Millennium does not alter Steam memory and runs completely disjoint.
