diff --git a/src/sys/env.cc b/src/sys/env.cc
index 20f31970..ae578d4c 100644
--- a/src/sys/env.cc
+++ b/src/sys/env.cc
@@ -109,7 +109,7 @@ const void SetupEnvironmentVariables()
         #ifdef _WIN32
             const auto shimsPath = SystemIO::GetInstallPath().string() + "/ext/data/shims";
         #elif __linux__
-            const auto shimsPath = "/usr/share/millennium/shims";
+            const auto shimsPath = "@shims@/share/millennium/shims";
         #endif
     #endif
 
@@ -120,7 +120,7 @@ const void SetupEnvironmentVariables()
         #ifdef _WIN32
             const auto assetsPath = SystemIO::GetInstallPath().string() + "/ext/data/assets";
         #elif __linux__
-            const auto assetsPath = "/usr/share/millennium/assets";
+            const auto assetsPath = "@assets@/share/millennium/assets";
         #endif
     #endif
 
@@ -143,15 +143,11 @@ const void SetupEnvironmentVariables()
     const std::string configDir = GetEnvWithFallback("XDG_CONFIG_HOME", fmt::format("{}/.config", homeDir));
     const std::string dataDir   = GetEnvWithFallback("XDG_DATA_HOME", fmt::format("{}/.local/share", homeDir));
     const std::string stateDir  = GetEnvWithFallback("XDG_STATE_HOME", fmt::format("{}/.local/state", homeDir));
-    const static std::string pythonEnv = fmt::format("{}/millennium/.venv", dataDir);
+    const static std::string pythonEnv = "@venv@";
     const std::string pythonEnvBin = fmt::format("{}/bin/python3.11", pythonEnv);
-
-    if (access(pythonEnvBin.c_str(), F_OK) == -1) {
-        std::system(fmt::format("{}/bin/python3.11 -m venv {} --system-site-packages --symlinks", MILLENNIUM__PYTHON_ENV, pythonEnv).c_str());
-    }
   
     std::map<std::string, std::string> environment_unix = {
-        { "MILLENNIUM_RUNTIME_PATH", "/usr/lib/millennium/libmillennium_x86.so" },
+        { "MILLENNIUM_RUNTIME_PATH", "@OUT@/lib/millennium/libmillennium_x86.so" },
         { "LIBPYTHON_RUNTIME_PATH",  LIBPYTHON_RUNTIME_PATH },
 
         { "MILLENNIUM__STEAM_EXE_PATH", fmt::format("{}/.steam/steam/ubuntu12_32/steam",     homeDir) },
