diff --git a/src/sys/env.cc b/src/sys/env.cc
index 20f31970..c3e6a1ab 100644
--- a/src/sys/env.cc
+++ b/src/sys/env.cc
@@ -109,7 +109,7 @@ const void SetupEnvironmentVariables()
         #ifdef _WIN32
             const auto shimsPath = SystemIO::GetInstallPath().string() + "/ext/data/shims";
         #elif __linux__
-            const auto shimsPath = "/usr/share/millennium/shims";
+            const auto shimsPath = "@shims@/share/millennium/shims";
         #endif
     #endif
 
@@ -120,7 +120,7 @@ const void SetupEnvironmentVariables()
         #ifdef _WIN32
             const auto assetsPath = SystemIO::GetInstallPath().string() + "/ext/data/assets";
         #elif __linux__
-            const auto assetsPath = "/usr/share/millennium/assets";
+            const auto assetsPath = "@assets@/share/millennium/assets";
         #endif
     #endif
 
@@ -151,7 +151,7 @@ const void SetupEnvironmentVariables()
     }
   
     std::map<std::string, std::string> environment_unix = {
-        { "MILLENNIUM_RUNTIME_PATH", "/usr/lib/millennium/libmillennium_x86.so" },
+        { "MILLENNIUM_RUNTIME_PATH", "@OUT@/lib/millennium/libmillennium_x86.so" },
         { "LIBPYTHON_RUNTIME_PATH",  LIBPYTHON_RUNTIME_PATH },
 
         { "MILLENNIUM__STEAM_EXE_PATH", fmt::format("{}/.steam/steam/ubuntu12_32/steam",     homeDir) },
